tests/Utilities/ImageAssetTest.php

```diff
@@ -11,18 +11,10 @@ class ImageAssetTest extends TestCase
 {
     protected const DATA_IMAGES = __DIR__ . DS;
 
-    protected function setUp(): void
-    {
-        parent::setUp();
-        if (!defined('PLUGIN_ROOT')) {
-            include __DIR__ . DS . '..' . DS . 'bootstrap.php';
```

Comment:
I can't run tests from the phpstorm sidebar without this as the bootstrap script seems not to be run then.

Reply:
This is a bug in PHPStrom.
You can manually tell PHPStorm to use `phpunit.xml.dist` like so

<img width="1301" alt="image" src="https://github.com/brandcom/cakephp-assets/assets/9105243/648d4880-9932-44d9-80c3-b451228bab26">

Reply:
Ah... now it works :) 

![Mittel (Bildschirmfoto 2023-09-23 um 21 15 33)](https://github.com/brandcom/cakephp-assets/assets/77938819/6bf2eeac-f722-4f0e-b13d-dce5069ffd90)